keyboards/cospad/cospad.h

```diff
@@ -14,12 +13,14 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
+
 #pragma once
 
 #include "quantum.h"
-#define ___ KC_NO
 
-/* This a shortcut to help you visually see your layout.
+#define XXX KC_NO
```

Comment:
There are plenty of boards which use the `___` style, is there any reason to change this?

Reply:
I'd rather keep these 3-character versions consistent with the 7-character versions.

Reply:
There are 49 instances of `___` vs 64 for `xxx` or `XXX` and 9 for `_x_`.

While I agree with the case of consistency, I personally find `___` easier to read within the context of the LAYOUT macros.